network-api/networkapi/utility/middleware.py

```diff
@@ -37,6 +37,7 @@ def __call__(self, request):
                     redirect_url = '{protocol}://{hostname}{path}'.format(
                         protocol=protocol,
                         hostname=hostnames[0],
+                        # FIXME:TODO: what to do with this /en link
```

Comment:
not 100% sure if we need to keep the `/en` here

Reply:
yeah, this shouldn't need the `/en` prefix, it should automatically add that in.